### ISSUE TITLE
Collaborators wanted

### ISSUE THREAD

**Issue by jeaye** [jeaye/ncurses-rs#102]
I spend less and less time with Rust these days, so it's been up to the community to keep ncurses-rs going. Fortunately, you guys are awesome. I'd like to bring on one or two collaborators, who have committed to ncurses-rs already, or who're prominent enough in the Rust scene, to help make sure ncurses-rs is going where it should.

If you'd like to volunteer, comment here or shoot me an email if you'd prefer to keep it private.

**Comment by mdenchev** [comment]
I wouldn't mind contributing if you are still looking. I also like the idea of adding a more idiomatic Rust interface on top, either in a separate module or a separate crate.

**Comment by jeaye** [comment]
Absolutely. Rust needs a good ncurses wrapper and I haven't been using Rust much these days. If you're interested in improving the API, we can certainly chat more about it here and on IRC and you're welcome to get started with some PRs.

I'd like to see ncurses-rs progress to something more than a thin C wrapper; any sort of iterative process to find a suitable API, likely in a separate module, would be great.

**Comment by mdenchev** [comment]
Cool. I'm new to ncurses but on a first pass the following come to mind: 
 - create a struct for WINDOW and move all w* functions into it. This would get rid of newwin and delwin. stdscr seems to be also a WINDOW so the other methods that act on it implicitly can be made into method calls instead
- make the various get* functions return values instead of take mut refs
- encapsulate mouse input, though I haven't played with those functions yet so no exact ideas here yet
- add resizing logic

I've played with ncurses for 2 days now so I'm sure some of the ideas would need to be rethought :). Thoughts?
